dex/networks/polygon/params.go

```diff
@@ -36,9 +36,9 @@ var (
 
 	ContractAddresses = map[uint32]map[dex.Network]common.Address{
 		0: {
-			// dex.Mainnet: common.HexToAddress(""),
-			// dex.Testnet: common.HexToAddress(""),
-			dex.Simnet: common.HexToAddress(""), // Filled in by MaybeReadSimnetAddrs
+			dex.Mainnet: common.HexToAddress("0xd45e648D97Beb2ee0045E5e91d1C2C751Cd0Bc00"), // txid: 0xbb7d09fb3832b35fbbed641453a90f217a2736cf1419848887dfee2dbb14187e
```

Comment:
Confirmed.  https://polygonscan.com/address/0xd45e648D97Beb2ee0045E5e91d1C2C751Cd0Bc00
Will you please attempt to "Verify and Publish" the Solidity source with Polyscan?  If we can't get it verified, it might be a discrepancy with the optimization settings we used to generate the bytecode in the repo, whereas we'd want 200 iterations of optimization.

Reply:
Done